.install/debian_gnu_linux_12.sh

```diff
@@ -0,0 +1,22 @@
+
+#!/bin/bash
+ARCH=$(uname -m)
+export DEBIAN_FRONTEND=noninteractive
+MODE=$1 # whether to install using sudo or not
+
+$MODE apt update -qq
+$MODE apt install -yqq git wget build-essential lcov openssl libssl-dev \
+        python3 python3-venv python3-dev rsync unzip
+
+# Install gcc-11 and g++-11 because gcc-12.2 fails to compile VectorSimilarity
+# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105593
+if [[ $ARCH = 'x86_64' ]]
+then
+        $MODE apt install -yqq gcc-11 g++-11
+        # Update alternatives
+        $MODE update-alternatives \
+                --install /usr/bin/gcc gcc /usr/bin/gcc-11 60 \
+                --slave /usr/bin/g++ g++ /usr/bin/g++-11
+fi
```

Comment:
can we address Meirav's message and use GCC 12.3/13? attempting to update GCC with `apt install -yqq gcc` or `apt upgrade gcc` doesn't fetch 12.3?

Reply:
apt does not fetch 12.3, I tried with these repositories, but none of them contains 12.3:
[bookworm](https://packages.debian.org/bookworm/)
[bookworm-updates](https://packages.debian.org/bookworm-updates/)
[bookworm-backports](https://packages.debian.org/bookworm-backports/)
Should I use a different repository?

Reply:
Could [this](https://packages.debian.org/trixie/amd64/gcc/download) be relevant?

Reply:
Possibly [this](https://hub.docker.com/layers/library/gcc/12.4.0-bookworm/images/sha256-1f21186263c2a5823c0ee81033d2980256a958af8a53801e7baf5744e07ff80f?context=explore) as well

Reply:
We can also try setting a make/cmake arg that will silent the warning (specifically for this OS on CI), or add back the ignore flag to vecsim cmake